contrib/devtools/update-translations.py

```diff
@@ -80,6 +82,10 @@ def sanitize_string(s):
     return s.replace('\n',' ')
 
 def check_format_specifiers(source, translation, errors, numerus):
+    # Check for btc address in translation after removing whitespaces
+    if re.search(ADDRESS_RE, ''.join(translation.split())):
+        errors.append("WARNING...Address found in translation for '%s': '%s'\nMessage will be deleted" % (sanitize_string(source), sanitize_string(translation)))
```

Comment:
There's not need to add the "\nMessage will be deleted" here, it's the case for all errors so doesn't need mentioning.